### PR TITLE
Increase Reports test coverage

### PR DESCRIPTION
**Pull request by reinbach**
Attempt to increase test coverage in the reports app

See https://github.com/unicef/etools-issues/issues/1106

### CODE REVIEW

File: EquiTrack/reports/tests/test_models.py [hunk 1]
```diff
@@ -90,3 +108,168 @@ def test_indicator(self):
...
+        """Check that all future country programmes have
+        from date in the future
+        """
+        self.assertItemsEqual(
+            [x.pk for x in CountryProgramme.objects.all_future],
+            [self.programme_future.pk]
+        )
+
+    def test_all_active_and_future(self):
+        """Check that all future and actuve country programmes have
```
Comment (dpoirier) [inline EquiTrack/reports/tests/test_models.py]: s/actuve/active/

File: EquiTrack/reports/tests/test_models.py [hunk 2]
```diff
@@ -90,3 +108,168 @@ def test_indicator(self):
...
+        to date in the future
+        """
+        self.assertItemsEqual(
+            [x.pk for x in CountryProgramme.objects.all_active_and_future],
+            [self.programme_future.pk, self.programme_active.pk]
+        )
+
+    def test_expired(self):
+        """Check that future country programmes have
+        from date in the future
```
Comment (dpoirier) [inline EquiTrack/reports/tests/test_models.py]: Looks like the docstring from test_future()?

File: EquiTrack/reports/tests/test_models.py [hunk 3]
```diff
@@ -90,3 +108,168 @@ def test_indicator(self):
...
+        """If 'A0/99' in wbs mark country programme as invalid"""
+        programme = CountryProgrammeFactory()
+        self.assertFalse(programme.invalid)
+        programme.wbs = "A0/99"
+        programme.save()
+        self.assertTrue(programme.invalid)
+
+    def test_save_to_date(self):
+        """If to_date changes for country programme agreement
+        to date needs to be updated as well
```
Comment (dpoirier) [inline EquiTrack/reports/tests/test_models.py]: Adding a comma after "programme" would make this docstring easier to parse.

File: EquiTrack/reports/views/v2.py [hunk 4]
```diff
@@ -121,7 +121,7 @@ def list(self, request, pk=None, format=None):
         """
         Return All Indicators for Result
         """
-        indicators = Indicator.objects.filter(result_id=pk)
+        indicators = Indicator.objects.filter(result__pk=pk)
```
Comment (dpoirier) [inline EquiTrack/reports/views/v2.py]: Why this change? It looks equivalent.

Comment (reinbach) [reply EquiTrack/reports/views/v2.py]: It is equivalent, but I have a preference for the `__pk` as it is clearer to me that we're working with primary key of `result`, and not a field that happens to end in `_id` :)

**Comment by robertavram** [comment]
tests are failing after updates... could you fix them?

**Comment by ntrncic** [comment]
please resolve conflicts.. sorry

**Comment by ntrncic** [comment]
@reinbach please resolve conflicts.
